files/base.rb

```diff
@@ -67,11 +67,15 @@ def tip
     @event['check']['tip']
   end
 
-  def description(maxlen=100000)
-    description = @event['check']['notification']
+  def client_display_name
     client_display_name = @event['client']['tags']['Display Name'] rescue nil
     client_display_name = @event['client']['name'] if
       client_display_name.nil? || client_display_name.empty?
+    client_display_name
+  end
+
+  def description(maxlen=100000)
+    description = @event['check']['notification']
     description ||= [client_display_name, @event['check']['name'], uncolorize(@event['check']['output'])].join(' : ')
```

Comment:
Do you think it's valuable to keep client_display_name in the description as well?